ci/setup-rust.sh

```diff
@@ -0,0 +1,14 @@
+#!bash
+set -eux
+
+RUST_TOOLCHAIN=${1:-1.62}
```

Comment:
Can you add a check to avoid the curl + bash if rust toolchain is already installed.

- this is useful when offline or with poor connection
- curl + bash exec is always a security hazard, so better avoiding it when not strictly needed

Reply:
done